.travis.yml

```diff
@@ -1,21 +1,25 @@
 language: python
+sudo: false
 python:
   - "2.7"
   - "3.4"
+addons:
+  apt:
+    packages:
+    - libblas-dev
+    - liblapack-dev
+    - gfortran
 before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq libatlas-dev libatlas-base-dev liblapack-dev gfortran
```

Comment:
`libatlas-dev` and `libatlas-base-dev` are replaced by `libblas-dev`? That would be a slower BLAS library, wouldn't it? What about `libopenblas-dev`?